### PR TITLE
fix: refactor filter sidebar to be glimmer component

### PR DESCRIPTION
**Pull request by MitanOmar**

### CODE REVIEW

**Comment by derrabauke** [comment]
`filter-sidebar/label` component is still `classic`. Please add here.

File: app/styles/app.scss [hunk 1]
```diff
@@ -43,6 +43,8 @@
 @import "components/sy-toggle";
 @import "components/progress-tooltip";
 @import "components/welcome-modal";
+@import "components/loading-icon";
+@import "components/filter-sidebar";
```
Comment (derrabauke) [inline app/styles/app.scss:47]: Update path according to filename.

File: app/styles/components/filter-sidebar.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
+.filter-sidebar-group {
+  display: flex;
+  flex-direction: column;
+}
+
+.filter-sidebar-group--expanded .filter-sidebar-group-label .fa {
```
Comment (derrabauke) [inline app/styles/components/filter-sidebar.scss:6]: Be aware that you made the css selector less accurate while deleting `&.` in front of it. The exact replacement would have been:
```scss
.filter-sidebar-group .filter-sidebar-group--expanded .filter-sidebar-group-label .fa
``` 

or indent the those rules in the parenting block `.filter-sidebar-group { }`

File: app/styles/app.scss [hunk 3]
```diff
@@ -43,6 +43,8 @@
 @import "components/sy-toggle";
 @import "components/progress-tooltip";
 @import "components/welcome-modal";
+@import "components/loading-icon";
```
[the remark this answers is not in the document]
Comment (MitanOmar) [reply app/styles/app.scss:46]: it come by a mistake, and resolved it

File: app/styles/components/filter-sidebar--groups.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
+.filter-sidebar-group {
+  display: flex;
+  flex-direction: column;
+
+  & {
+    display: flex;
+    flex-direction: column;
+  }
```
Comment (derrabauke) [inline app/styles/components/filter-sidebar--groups.scss:8]: This is duplicated now. :)

File: app/styles/app.scss [hunk 5]
```diff
@@ -43,6 +43,7 @@
 @import "components/sy-toggle";
 @import "components/progress-tooltip";
 @import "components/welcome-modal";
+@import "components/filter-sidebar--groups";
```
Comment (derrabauke) [inline app/styles/app.scss:46]: nitpick:
```suggestion
@import "components/filter-sidebar--group";
```

**Review by derrabauke — APPROVED** [review]

**Pull request merged**